test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Great job adding the file `test.md` 👌, but according to the requirements you should ignore this file by git, please this filet to `.gitignore` file and make sure that the file is not present in your GitHub repository.

![1](https://user-images.githubusercontent.com/45231276/174903490-ccca7e70-b64b-4348-a3a6-2f8263bdcace.PNG)